go/vt/vtgate/buffer/shard_buffer.go

```diff
@@ -25,13 +25,14 @@ import (
 	"time"
 
 	"vitess.io/vitess/go/vt/discovery"
+
+	"vitess.io/vitess/go/vt/vtgate/errorsanitizer"
+
 	"vitess.io/vitess/go/vt/log"
```

Comment:
nit: import format

Reply:
fixed via a43a4d49909, we don't want this change